sw/device/silicon_creator/manuf/data/manuf_testplan.hjson

```diff
@@ -0,0 +1,220 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+{
+  name: "manufacturing"
+
+  testpoints: [
+    {
+      name: manuf_cp_unlock_raw
+      desc: '''Verify transition from RAW to TEST_UNLOCKED lc_state.
+
+            - Pre-load OTP with RAW lc_state.
+            - Bring device out of reset selecting the Life Cycle (LC) TAP interface using strap pins.
+
+            Perfom the following steps via LC TAP interface:
+
+            - Provide RAW_UNLOCK.
+            - Poll for lc_ctrl STATUS register for TRANSITION_SUCCESSFUL. Check for errors or timeout.
+            - Perform reset by toggling the POR pin.
+            - Verify the lc_state is TEST_UNLOCKED0.
+
+            Repeat the test case with TRANSITION_CTRL.EXT_CLOCK_EN set.
```

Comment:
this one is a good point... we might want to do this one always with EXT_CLOCK_EN, otherwise the raw silicon clock might just be too off.

Reply:
yeah for RAW unlock specifically, my expectation is that we will always use EXT CLOCK.